boards/arm/thingy53_nrf5340/Kconfig.defconfig

```diff
@@ -82,6 +82,25 @@ config REGULATOR
 
 endif # !TRUSTED_EXECUTION_SECURE
 
+if BOARD_ENABLE_USB_CDC
+
+config USB_DEVICE_STACK
```

Comment:
We could set the following to prevent flooding USB CDC ACM with the warning logs observed by @kapi-no:
```
CONFIG_USB_CDC_ACM_LOG_LEVEL_ERR=y
CONFIG_USB_DEVICE_LOG_LEVEL_ERR=y
```

Reply:
Fixed